spec/jobs/performance_platform_transactions_queue_job_spec.rb

```diff
@@ -25,22 +16,10 @@
   end
 
   it 'enqueues a job to send all published job from yesterday' do
-    stub_const('PP_TRANSACTIONS_BY_CHANNEL_TOKEN', 'not-nil')
-
-    pp = double(:performance_platform)
-    two_days_ago = Date.current.beginning_of_day.in_time_zone - 2.days
-    today = Date.current.beginning_of_day.in_time_zone
-
-    build(:vacancy, :past_publish, publish_on: two_days_ago).save(validate: false)
-    build(:vacancy, :past_publish, publish_on: today).save(validate: false)
-
-    jobs_published_yesterday = [
-      build(:vacancy, :past_publish, publish_on: date_to_upload).save(validate: false),
-      build(:vacancy, :past_publish, publish_on: date_to_upload).save(validate: false)
-    ]
+    pp = instance_double(PerformancePlatformSender::Transactions)
 
-    expect(PerformancePlatform::TransactionsByChannel).to receive(:new).with('not-nil').and_return(pp)
-    expect(pp).to receive(:submit_transactions).with(jobs_published_yesterday.count, date_to_upload.iso8601)
```

Comment:
Have we lost the test of this functionality?

Reply:
For the job classes yes. These specific tests have now moved into the `PerformancePlatformSender` class spec instead.

Reply:
My read of those suggests that we're no longer testing that the data is collected correctly in those tests.

Reply:
You're right @erbridge - just pushed a commit to fix this. Nice catch 👍